core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -25,8 +25,8 @@ DELTA_JSON = {
                 'bind': '/etc/dhcpcd.conf',
                 'mode': 'rw'
             },
-            "/usr/blueos/userdata": {
```

Comment:
I suspect this could cause some setups to end up with both `/usr/blueos/userdata` and `/usr/blueos`  bindings. can you check?